### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by martinwrudolf**
Better README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,75 @@
 # InstaTonne
-distributed social media app for CMPUT404 :)
+[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature. Our aim is to provide users with more autonomy and choice while connecting them to various independent social networks without being limited to a single platform like Facebook, Instagram or Twitter.
```
Comment (rydengraham) [inline README.md:2]: maybe change to this:
that embraces the interconnectedness and peer-to-peer nature of the web.

Comment (martinwrudolf) [reply README.md:2]: kk

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # InstaTonne
-[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature. Our aim is to provide users with more autonomy and choice while connecting them to various independent social networks without being limited to a single platform like Facebook, Instagram or Twitter.
+[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature.
```
Comment (rydengraham) [inline README.md:2]: Sorry, I should have clarified; the references to other platforms are fine, I mostly wanted the grammatical error fixed

**Review by martinwrudolf — COMMENTED** [review]
updated

**Review by rydengraham — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (martinwrudolf) [reply README.md]: Did I resolve the grammar error? Cause I really don't know what grammar error you were talking about. Let me know if there's still something needing fixing

**Pull request merged**